6.6/Dockerfile

```diff
@@ -76,11 +76,19 @@ RUN set -e; \
   chown -R "$SOLR_USER:$SOLR_GROUP" /opt/solr /docker-entrypoint-initdb.d /opt/docker-solr && \
   chown -R "$SOLR_USER:$SOLR_GROUP" /opt/mysolrhome
 
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static /tini-static
```

Comment:
Follow https://github.com/docker-library/official-images/blob/master/README.md#init and put it in /usr/local/bin

Reply:
Done. But I've adjusted the command to fit better in our existing gpg handling. Hope that is ok.